### PR TITLE
Allow releasing multiple reservations at once

### PR DESCRIPTION
**Pull request by mlaitinen**
Fixes issue #50

### CODE REVIEW

File: stock_reserve/model/stock_reserve.py [hunk 1]
```diff
@@ -132,8 +132,9 @@ def release(self):
         """
         Release moves from reservation
         """
-        move_recs = self.move_id
-        move_recs.action_cancel()
+        for rec in self:
```
Comment (pedrobaeza) [inline stock_reserve/model/stock_reserve.py]: You can do better: `self.mapped('move_id').action_cancel()`

Comment (mlaitinen) [inline stock_reserve/model/stock_reserve.py]: Awesome, thanks! I learned about `mapped()` only after I had originally made this change.

**Comment by guewen** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**